components/director/internal/domain/formation/service.go

```diff
@@ -1769,3 +1841,45 @@ func isObjectTypeSupported(formationTemplate *model.FormationTemplate, objectTyp
 
 	return true
 }
+
+// updateFormationAssignmentsWithStateForObjectID lists all formation assignments for a given objectID that are either source or target of the assignment
```

Comment:
Can we mention in the comment that the function also resets the config and error of the FAs